manifests/t/Tyrrrz/LightBulb/2.3.3/Tyrrrz.LightBulb.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Tyrrrz.LightBulb
+PackageVersion: 2.3.3
+InstallerType: exe

```

```suggestion
InstallerType: inno
```
🤦‍♂️ didn't see that one